kubetools/kubernetes/config/__init__.py

```diff
@@ -353,6 +353,8 @@ def generate_kubernetes_configs_for_project(
         schedule = cronjob['schedule']
         concurrency_policy = cronjob['concurrency_policy']
         batch_api_version = cronjob.get('batch-api-version')  # May depend on target cluster
+        successfulJobsHistoryLimit = cronjob.get('successfulJobsHistoryLimit', 3)
+        failedJobsHistoryLimit = cronjob.get('failedJobsHistoryLimit', 1)
```

Comment:
If there's already a default value in k8s, we should not override it here but rely on it instead. So basically we should not have a default value here.

Reply:
https://github.com/EDITD/kubetools/pull/166/commits/b60c09557b9a04a5ba8f7c486d8b8c36d424adb0